apps/server/src/modules/tool/service/context-external-tool.service.spec.ts

```diff
@@ -41,8 +42,10 @@ describe('ContextExternalToolService', () => {
 			};
 
 			it('should call find()', async () => {
-				const { schoolExternalToolId } = setup();
+				const { schoolExternalToolId, contextExternalTool1, contextExternalTool2 } = setup();
+
 				await service.deleteBySchoolExternalToolId(schoolExternalToolId);
+				contextExternalToolRepo.find.mockResolvedValue([contextExternalTool1, contextExternalTool2]);
```

Comment:
Why this change?

Reply:
missing test coverage (that was before the new SonarCloud test coverage )